yb-voyager/cmd/exportSchema.go

```diff
@@ -147,7 +147,11 @@ func clearSchemaIsExported() {
 }
 
 func updateIndexesInfoInMetaDB() {
-	log.Infof("updating indexes info in meta db")
+	log.Infof("updating indexes info in metaDB")
+	if !utils.ContainsString(source.ExportObjectTypesList, "TABLE") {
```

Comment:
What if the user has already migrated TABLEs using some previous invocation?

Reply:
I didn't get your question.
If TABLEs are already exported in the previous invocation then the INDEXES dependent on them would also have been exported in that invocation itself.